homeassistant/components/vicare/__init__.py

```diff
@@ -109,5 +109,13 @@ def get_supported_devices(
     return [
         device_config
         for device_config in devices
-        if device_config.getModel() not in ["Heatbox1", "Heatbox2_SRC"]
+        if device_config.getModel()
+        not in [
+            "Heatbox1",
+            "Heatbox2_SRC",
+            "E3_FloorHeatingCircuitChannel",
+            "E3_FloorHeatingCircuitDistributorBox",
+            "E3_RoomControl_One_522",
+            "E3_RoomSensor",
+        ]
```

Comment:
If this is called multiple times you can do a tiny improvement by statically define this as a constant

Reply:
thanks, move to `const.py`